src/components/inputs/text/text.component.tsx

```diff
@@ -19,6 +19,7 @@ const TextField: React.FC<FormFieldProps> = ({ question, onChange, handler, prev
   const [errors, setErrors] = useState([]);
   const [warnings, setWarnings] = useState([]);
   const isFieldRequiredError = useMemo(() => errors[0]?.errCode == fieldRequiredErrCode, [errors]);
+  const isFieldConditionalRequiredErrCode = useMemo(() => errors[0]?.errCode == fieldConditionalRequiredErrCode, [errors]);
 
```

Comment:
Can we use a component wrapper handling the error case, since we have to the same at multiple places?

Reply:
Done